kong/plugins/rate-limiting/handler.lua

```diff
@@ -25,17 +25,21 @@ RateLimitingHandler.VERSION = "2.0.0"
 local function get_identifier(conf)
   local identifier
 
-  if conf.limit_by == "consumer" then
+  if conf.limit_by == "service" then
+    identifier = ""
```

Comment:
shouldn't this be the service id? if using a constant then there will be 1 global rate limit no?

Reply:
the get_local_key method in init.lua had concat service id and route id(if existed)

`local get_local_key = function(conf, identifier, period, period_date)
  local service_id, route_id = get_service_and_route_ids(conf)

  return fmt("ratelimit:%s:%s:%s:%s:%s", route_id, service_id, identifier,
             period_date, period)
end`

Reply:
yes, that is correct. But please add a test that validates the behaviour anyway.